echopype/utils/io.py

```diff
@@ -424,3 +433,74 @@ def validate_source_ds_da(
         check_file_existence(file_path=source_ds_da, storage_options=storage_options)
 
     return source_ds_da, file_type
+
+
+# Utilities for creating temporary swap zarr files -------------------------------------
+def _create_zarr_store_map(path: str, storage_options: dict) -> FSMap:
```

Comment:
I couldn't find any use of this in the code base, did I miss anything?

Reply:
You're right! Thanks for catching that. This was an old function. Just cleaned up.